apps/web/src/pages/Users/UserInformationPage/components/AboutSection.tsx

```diff
@@ -36,11 +37,7 @@ const AboutSection = ({ user }: BasicUserInformationProps) => {
         </div>
         <div data-h2-flex-item="base(1of1) p-tablet(1of2) desktop(1of3)">
           <p data-h2-font-weight="base(700)">
-            {intl.formatMessage({
-              defaultMessage: "Preferred spoken interview language",
-              id: "DB9pFd",
-              description: "Title for preferred spoken interview language",
-            })}
+            {}
```

Comment:
Oops, you deleted that

![image](https://github.com/user-attachments/assets/983605db-cef5-49c4-9afe-639fbc775025)

Reply:
Thank you :+1: Added back in [Add back label for spoken interview language](https://github.com/GCTC-NTGC/gc-digital-talent/pull/13300/commits/84b3899cfa02247a2e9b55a5e5e61cb85319948a)